### DISCUSSION TITLE
Add local time for models.

### DISCUSSION THREAD

**Discussion by roelfrenkema** [open-webui/open-webui#3460] [General]
**Is your feature request related to a problem? Please describe.**
Its frustrating to hear goodmorning when you go to bed.

**Describe the solution you'd like**
Add system time to Personalization would be the shortest way.

**Comment by atgehrhardt** [comment]
{{CURRENT_TIME}}

{{CURRENT_DATE}}

in the global system prompt or model system prompt

You can also get your username with {{USER_NAME}}
